sdk/core/azure-core/src/main/java/com/azure/core/http/rest/RequestOptions.java

```diff
@@ -125,13 +128,15 @@
     }
 
     /**
-     * Gets whether an exception is thrown when an HTTP response with a status code indicating an error
-     * (400 or above) is received.
+     * Gets the {@link ErrorOptions} that determines how error responses (400 or above) are handled.
+     * <p>
+     * Default is to throw.
      *
-     * @return true if to throw on status codes of 400 or above, false if not. Default is true.
+     * @return The {@link ErrorOptions} that determines how error responses (400 or above) are handled. Default is
+     * to throw.
      */
-    boolean isThrowOnError() {
```

Comment:
I assume it was never released, so we can just remove it?

Reply:
Correct

Reply:
Correct, this has never been GA'd